library/packages/test/file_conflict_callbacks_test.rb

```diff
@@ -24,6 +24,95 @@ def CallbackFileConflictReport(func)
   def CallbackFileConflictFinish(func)
     @fc_finish = func
   end
+
+  # run this command in "irb -ryast" to obtain the method names:
+  # Yast.import "Pkg"; Yast::Pkg.methods.select {|m| m.to_s.start_with?("Callback")}
+  # (Remove the methods which are defined above.)
+  MOCK_METHODS = [
+    :CallbackAcceptFileWithoutChecksum,
+    :CallbackAcceptUnknownDigest,
+    :CallbackAcceptUnknownGpgKey,
+    :CallbackAcceptUnsignedFile,
+    :CallbackAcceptVerificationFailed,
+    :CallbackAcceptWrongDigest,
+    :CallbackAuthentication,
+    :CallbackDestDownload,
+    :CallbackDoneDownload,
+    :CallbackDonePackage,
+    :CallbackDoneProvide,
+    :CallbackDoneRefresh,
+    :CallbackDoneScanDb,
+    :CallbackErrorScanDb,
+    :CallbackFinishDeltaApply,
+    :CallbackFinishDeltaDownload,
+    :CallbackImportGpgKey,
+    :CallbackInitDownload,
+    :CallbackMediaChange,
+    :CallbackMessage,
+    :CallbackNotifyConvertDb,
+    :CallbackNotifyRebuildDb,
+    :CallbackPkgGpgCheck,
+    :CallbackProblemDeltaApply,
+    :CallbackProblemDeltaDownload,
+    :CallbackProcessDone,
+    :CallbackProcessNextStage,
+    :CallbackProcessProgress,
+    :CallbackProcessStart,
+    :CallbackProgressConvertDb,
+    :CallbackProgressDeltaApply,
+    :CallbackProgressDeltaDownload,
+    :CallbackProgressDownload,
+    :CallbackProgressPackage,
+    :CallbackProgressProvide,
+    :CallbackProgressRebuildDb,
+    :CallbackProgressReportEnd,
+    :CallbackProgressReportProgress,
+    :CallbackProgressReportStart,
+    :CallbackProgressScanDb,
+    :CallbackResolvableReport,
+    :CallbackScriptFinish,
+    :CallbackScriptProblem,
+    :CallbackScriptProgress,
+    :CallbackScriptStart,
+    :CallbackSourceChange,
+    :CallbackSourceCreateDestroy,
+    :CallbackSourceCreateEnd,
+    :CallbackSourceCreateError,
+    :CallbackSourceCreateInit,
+    :CallbackSourceCreateProgress,
+    :CallbackSourceCreateStart,
+    :CallbackSourceProbeEnd,
+    :CallbackSourceProbeError,
+    :CallbackSourceProbeFailed,
+    :CallbackSourceProbeProgress,
+    :CallbackSourceProbeStart,
+    :CallbackSourceProbeSucceeded,
+    :CallbackSourceReportDestroy,
+    :CallbackSourceReportEnd,
+    :CallbackSourceReportError,
+    :CallbackSourceReportInit,
+    :CallbackSourceReportProgress,
+    :CallbackSourceReportStart,
+    :CallbackStartConvertDb,
+    :CallbackStartDeltaApply,
+    :CallbackStartDeltaDownload,
+    :CallbackStartDownload,
+    :CallbackStartPackage,
+    :CallbackStartProvide,
+    :CallbackStartRebuildDb,
+    :CallbackStartRefresh,
+    :CallbackStartScanDb,
+    :CallbackStopConvertDb,
+    :CallbackStopRebuildDb,
+    :CallbackTrustedKeyAdded,
+    :CallbackTrustedKeyRemoved
+  ].freeze
+
+  MOCK_METHODS.each do |method|
+    # mock empty methods with a single argument
+    send(:define_method, method) do |func|
```

Comment:
is send really needed? why simple `define_method` do not work here? so something like `define_method(method){ |f| }`